fluxcd-demo/helm/templates/deployment.yaml

```diff
@@ -0,0 +1,85 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: echoserver
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: busybox
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "fluxcd-demo.fullname" . }}-echoserver
+  namespace: echoserver
+  labels:
+    app: echoserver-deploy
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: echoserver
+  template:
+    metadata:
+      labels:
+        app: echoserver
+    spec:
+      initContainers:
+        - name: "sleepy"
+          image: busybox:1.36
```

Comment:
Probably best to use a full image path. I recall seeing a post about CRI-O requiring full path soon but I can't find the link